src/modules/auth/containers/LoginCard.js

```diff
@@ -207,18 +216,20 @@ function LoginCard(props) {
 				props.app.addAlert(
 					"danger",
 					t("LoginCard|The provided information is likely incorrect. The login has failed"), 30
-				);
-			} else if (e.response.status == 504) {
-				props.app.addAlert(
-					"danger",
-					t("LoginCard|Can't proceed due to connection problems"), 30
-				);
-			} else {
-				props.app.addAlert(
-					"danger",
-					`${t("LoginCard|Something went wrong")}. ${e?.response?.data?.message}`, 30
-				);
-			}
+					);
```

Comment:
@queuing4oranges please put the code back as where it was, you added some indentation which is redundant, also it aggravate the review a bit, since its not related to the relevant changes you were up to